lib/app.js

```diff
@@ -1,7 +1,22 @@
+require('dotenv').config();
 const express = require('express');
 const app = express();
+const request = require('superagent');
+const cors = require('cors');
 
 app.use(express.json());
+app.use(cors());
+
+app.get('/api/v1/weather', async(req, res) => {
+  try {
+    const URL = `https://api.weatherbit.io/v2.0/current?lat=${process.env.LAT}&lon=${process.env.LON}&key=${process.env.WEATHERBIT_API_KEY}&units=I`;
```

Comment:
I believe the lat and lon coordinates will probably have to come from somewhere other than process.env.